### ISSUE TITLE
The example as-is doesn't work and doesn't make sense

### ISSUE THREAD

**Issue by dcapeluto** [flashbots/web3-flashbots#41]
Firstly, greatly appreciate the effort in making flashbots available to the Python community. Ty!

Running the simply.py example as is as I see it, has 3 issues for me. One is logical - as mentioned elsewhere - in which rarely does one have the receiver's private key to sign. The second may be code related and the compiler throws an error in the bundle declaration (line 72) with some dictionary merge issue. Is this common? Lastly, the library doesn't seem to be updated to work with EIP1559 - requiring gasFee as opposed to their new counterparts, which is not a work-stopper, but it's sub-optimal. 

Is it expected for this library to be maintained further?

(If I had the knowledge to contribute and further this library, I would)

Thanks again

**Comment by tksoftw** [comment]
Can confirm the example does NOT work at all. Do you have any updates on this?

**Comment by lecoos** [comment]
I ran into the same problem, and confirm that the main reason for me is the _master_ branch of this repo does not update to support EIP1559.

If you have the same reason, please have a look at #44 and [lekhovitsky/web3-flashbots](https://github.com/lekhovitsky/web3-flashbots.git).

Thanks to all contributors, and hope this repo can be maintained continuously : )

**Comment by msqwer123** [comment]
I have the same problem. How can I solve it?

**Comment by andrewnagata** [comment]
Still struggling with simple.py example. Looks like #44 was merged.

I am getting a TransactionNotFound error upon receipt of the flashbots bundle return request. It appears that the Flashbot transaction never happens. Bundle is sent, but the transaction is never recorded it seems.

I am running a mev-geth node on goerli to test with.

The error trace:
```
Traceback (most recent call last):
  File "/Users/andrew/Documents/VSCode/Flashbots/minter_py/example.py", line 111, in <module>
    receipts = result.receipts()
  File "/Users/andrew/Documents/VSCode/Flashbots/minter_py/flashbots/flashbots.py", line 72, in receipts
    return list(
  File "/Users/andrew/Documents/VSCode/Flashbots/minter_py/flashbots/flashbots.py", line 73, in <lambda>
    map(lambda tx: self.w3.eth.get_transaction_receipt(tx["hash"]), self.bundle)
  File "/usr/local/lib/python3.9/site-packages/web3/module.py", line 57, in caller
    result = w3.manager.request_blocking(method_str,
  File "/usr/local/lib/python3.9/site-packages/web3/manager.py", line 198, in request_blocking
    return self.formatted_response(response,
  File "/usr/local/lib/python3.9/site-packages/web3/manager.py", line 177, in formatted_response
    apply_null_result_formatters(null_result_formatters, response, params)
  File "/usr/local/lib/python3.9/site-packages/web3/manager.py", line 82, in apply_null_result_formatters
    formatted_resp = pipe(params, null_result_formatters)
  File "cytoolz/functoolz.pyx", line 667, in cytoolz.functoolz.pipe
  File "cytoolz/functoolz.pyx", line 642, in cytoolz.functoolz.c_pipe
  File "/usr/local/lib/python3.9/site-packages/web3/_utils/method_formatters.py", line 656, in raise_transaction_not_found
    raise TransactionNotFound(message)
web3.exceptions.TransactionNotFound: Transaction with hash: '0x5a7e6b317b297a477d2666ad318d4323745612ef67d33b3ca1d62f6b8f6e5a9a' not found.
```

**Comment by andrewnagata** [comment]
solved my problem... If you are using Goerli testnet, you need to set the relay uri when you create the Flashbot module. Otherwise, the default uri is set to the mainnet relay.

(around line 26)
```
w3 = Web3(HTTPProvider("http://10.0.1.68:8545"))
flashbot(w3, FLASHBOTS_SIGNATURE, "https://relay-goerli.flashbots.net")
```

**Comment by zeroXbrock** [comment]
resolved with #55

**Comment by zeroXbrock** [comment]
also note: EIP-1559 support was added in version 1.0.1
